planet/src/main/webapp/places.js

```diff
@@ -315,9 +342,32 @@ function savePlace(x) {
     if(x.innerHTML === "favorite_border") {
         // Set as saved
         x.innerHTML = "favorite";
+        // Save place to database
+        let name = document.getElementById('place-name').innerHTML; 
+        let address = document.getElementById('place-address').innerHTML; 
+        let open = document.getElementById('openingTime').innerHTML;
+        let close = document.getElementById('closingTime').innerHTML;
+        updateDatabase('test', name, address, open, close); 
     }
     else {
         // Set as unsaved
         x.innerHTML = "favorite_border";
+        // Delete place from database
+        deletePlace('test');
     }
 }
+
+/** Update database and add place with information */
+function updateDatabase(placeID, name, address, open, close) {
+    database.ref('places/' + placeID).set({
+        name: name,
+        address: address,
+        openingTime: open,
+        closingTime: close
+    });
+}
+
+/** Delete place from database when unsaved by user */
+function deletePlace(placeID) {
```

Comment:
if two users star a place, and then one unstars it, what should happen?

Reply:
Good point, the saved places should be separate and specific to each user's profile so I'll figure out how to implement that! I'm going to remove the "deletion portion" of the code and follow up with that in a separate PR